### PR TITLE
Page styling

### PR DESCRIPTION
**Pull request by amiraabouhadid**
Pull request for code review for project 1 of Microverse 2.0: 
Positioning and Floating Elements
We were asked to make a clone of a New York Times article page using the following concepts: Embedding images The box model Flexbox Positioning

Project specifications can be found at https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements

Project requirements: clean, code, semantic HTML, aside tag for editor's picks, use float, flex, and grid techniques.

Live Demo at https://amiraabouhadid.github.io/project1/
Author: Amira Abouhadid

Contributing reviews, issues, and feature requests are welcome!

Feel free to check the issues page.

Show your support Give a star if you like this project!

Acknowledgments to anyone whose code was used Inspiration etc. License This project is MIT licensed.

### CODE REVIEW

**Review by LamiaSristy — COMMENTED** [review]
# Status: Invalid Review Request ✖️  🔴 

Hello @amiraabouhadid.
**I can see a lot of effort and dedication here, but unfortunately, I have to mark this review request as invalid.** 😢 

## The issue:

**According to rules, all your GitHub Actions checks need to be successful, to be a valid CRR.**

**I have to mark this review request as invalid because one of your GitHub Actions `Linters / Webhint` checks was not successful.**

Please add linters to your project and then ask again for a CRR.

Feel free to reach me on slack @lamiasristy 🙋

Keep up the spirit always. Happy Coding 💻

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required 🛑

### Hello team 👋,

**Great job on this project 👏.**

### Highlights

- **The linters are properly configured.**
- **Proper use of flexbox and CSS grid.**

### Mandatory Requirements.

- **Remove the page horizontal scroll.**
- **Improve the design of the following sections so they look like the original.**
1.  The navbar.
2. The large image section. There should be social media icons between the image title and the image.
3.  Fix the Read 615 comments section. There should be social media icons next to the button. Also, the text above it should have a small font-size and a grey color.
4. In more in Space and Astronomy section, fix the width of the image captions. They should not have a greater width than the image. Also, add the small caption on the rightmost part below the image.
5. In Editor's pick section, add the small caption on the rightmost part below the image.
6. Fix the size of the new york times logo, below the Editor's pick, so it looks like the original.
7. In the footer subscribe section, the first four elements should have icons on the left; should have a different font-size and font-weight than the other elements.
- **You should not use a screenshot for the theory of inflation section. Use HTML and CSS to create it.**
- **It is a requirement that you use the float CSS property as well. See [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from pathwright.**

### Optional

- **Improve the look of your README file. Hint: You can use the Microverse [template](https://github.com/microverseinc/readme-template).**


### Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

File: index.html [hunk 1]
```diff
@@ -2,17 +2,16 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8">
   <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
```
Comment (GzimAsani) [inline index.html:5]: Give title more width

File: index.html [hunk 2]
```diff
@@ -54,56 +55,348 @@ <h1> Space Ripples Reveal Big Bang’s Smoking Gun</h1>
...
+      <p>Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the
+        cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.</p>
+      <p>So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.</p>
+      <p>We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.</p>
+      <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”</p>
+      <div id='end-body'>
+        <div class='cmmtbuttonslinks'>
+          <div>A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. <a
```
Comment (GzimAsani) [inline index.html]: This div should have a space with with the button below it

File: index.html [hunk 3]
```diff
@@ -54,56 +55,348 @@ <h1> Space Ripples Reveal Big Bang’s Smoking Gun</h1>
...
+              href="https://www.parsintl.com/publication/the-new-york-times/">Order Reprints</a> | <a href="https://www.nytimes.com/section/todayspaper">Today’s Paper </a>| <a href="https://www.nytimes.com/subscription?campaignId=48JQY">Subscribe</a>
+          </div>
+          <button type="button" name="button">Read 615 Comments</button>
+          <div id=slinks>
+            <i class="fab fa-facebook-f"></i>
+            <i class="fab fa-twitter"></i>
+            <i class="fas fa-envelope"></i>
+            <i class="fas fa-share"></i>
+            <i class="fas fa-bookmark"></i>
+            <i class='fas fa-comment'></i>
```
Comment (GzimAsani) [inline index.html]: Those icons should be on the right side of the button not below it

File: index.html [hunk 4]
```diff
@@ -54,56 +55,348 @@ <h1> Space Ripples Reveal Big Bang’s Smoking Gun</h1>
...
+          </div>
+          <div class="nl-2-a">
+            <a href="#">See the latest</a>
+          </div>
+          <div class="nl-2-a">
+            <a href="#">See the latest</a>
+          </div>
         </div>
+        <hr>
+        <h3>More in Space and Astronomy</h3>
```
Comment (GzimAsani) [inline index.html:293]: This h3 should be on the same vertical line with the pictures below.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:
Hello **Team** 👋 Good job implementing so far :muscle:, however there are some more changes that you have to implement

Changes to be made:
- [ ] You are using repated css code like _display: flex_ 15 times and display: grid 6 times this is bad CSS practice what you can do is create a class like `.grid { display: grid; }` and use it every time you need
- [ ] The left and right elements of navbar should have some margins currently they are extending all the way on their sides
- [ OPTIONAL] Your readme could use some work, you can use the template that was provided by the previous TSE 
- [ ] Check the comments below for more requirements




### **Please make the required changes and submit another review request when you are ready.**

<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

**Comment by amiraabouhadid** [comment]
THANKS FOR THE REVIEWS. Opening a new pull request to test the linters.

**Pull request closed without merging**